src/drive/mobile/modules/settings/components/SelectMediaBuckets.jsx

```diff
@@ -0,0 +1,108 @@
+/* global cozy */
+import React from 'react'
+import { connect } from 'react-redux'
+import Main from 'drive/web/modules/layout/Main'
+import ScrollToTop from 'drive/web/modules/navigation/ScrollToTop'
+import styles from '../styles'
+import Toggle from 'cozy-ui/react/Toggle'
+import { getMediaBuckets, addMediaBucket, delMediaBucket } from '../duck'
+import { getMediaBuckets as getAllMediaBuckets } from 'drive/mobile/lib/media'
+
+const { BarCenter } = cozy.bar
+
+class SelectMediaBuckets extends React.Component {
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      mediaBuckets: []
+    }
+  }
+
+  createHandle(item) {
+    const self = this
```

Comment:
Is there a good reason to do that?  Can't we use this.props.add/delMediaBucket() directly?

Reply:
We can't use this.props.add/delMediaBucket() directly because the Toggle component use only one event with the state the component (boolean `checked`).